src/main/java/seedu/rc4hdb/logic/parser/FilterSpecifier.java

```diff
@@ -0,0 +1,21 @@
+package seedu.rc4hdb.logic.parser;
+
```

Comment:
We could probably move this class into a `filtercommands` package for better organisation.